cst438_gradebook/src/main/java/com/cst438/domain/EnrollmentDTO.java

```diff
@@ -2,4 +2,8 @@
 
 public record  EnrollmentDTO (int id, String studentEmail, String studentName, int courseId) {
 	
```

Comment:
This is not necessary, I am not sure how this would effect how the system works.